apps/web/package.json

```diff
@@ -16,7 +16,7 @@
     "@headlessui/react": "^1.7.19",
     "@heroicons/react": "^2.1.3",
     "@next/font": "^13.1.5",
-    "@rainbow-me/rainbowkit": "^2.1.2",
+    "@rainbow-me/rainbowkit": "2.1.3",
```

Comment:
do these need to be pinned, or can we keep the `^`?

Reply:
I think we can keep the ^. I don't know what components are reliant on earlier versions but smart wallet support shouldn't be affected by earlier versions.